src/main/java/com/iexec/common/docker/client/DockerClientFactory.java

```diff
@@ -47,7 +69,7 @@ public static synchronized DockerClientInstance getDockerClientInstance() {
      * @throws DockerException if the authentication fails
      */
     public static synchronized DockerClientInstance getDockerClientInstance(
-            String registryUrl, String username, String password) throws DockerException {
+            String registryUrl, String username, String password) throws Exception {
```

Comment:
Should check registryAddress?

Reply:
Actually, it is already checked inside `DockerClientInstance` constructor. It would be redundant here. I'll also remove the check in the other method [here](https://github.com/iExecBlockchainComputing/iexec-common/pull/249/files#diff-68d2a5a4981136e67af7a2df423f7cb358eac08e545c54a57268fdc04f4e70a4R51).

Reply:
:+1: